src/metrics/enhanced-metrics.ts

```diff
@@ -1,16 +1,61 @@
 import { getEnvValue, sendDistributionMetric } from "../index";
 
+import { Context } from "aws-lambda";
 import { parseTagsFromARN } from "../utils/arn";
 import { getColdStartTag } from "../utils/cold-start";
+import { getProcessVersion } from "../utils/process-version";
 
 const ENHANCED_LAMBDA_METRICS_NAMESPACE = "aws.lambda.enhanced";
 
-export function incrementInvocationsMetric(functionARN: string): void {
-  const tags = [...parseTagsFromARN(functionARN), getColdStartTag()];
-  sendDistributionMetric(`${ENHANCED_LAMBDA_METRICS_NAMESPACE}.invocations`, 1, ...tags);
+// Same tag strings added to normal Lambda integration metrics
+enum RuntimeTagValues {
+  Node8 = "nodejs8.10",
+  Node10 = "nodejs10.x",
 }
 
-export function incrementErrorsMetric(functionARN: string): void {
-  const tags = [...parseTagsFromARN(functionARN), getColdStartTag()];
-  sendDistributionMetric(`${ENHANCED_LAMBDA_METRICS_NAMESPACE}.errors`, 1, ...tags);
+/**
+ * Uses process.version to create a runtime tag
+ * If a version cannot be identified, returns null
+ * See https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html
+ */
+export function getRuntimeTag(): string | null {
+  const processVersion = getProcessVersion();
+  let processVersionTagString: string | null = null;
+
+  if (processVersion.startsWith("v8.10")) {
+    processVersionTagString = RuntimeTagValues.Node8;
+  }
+
+  if (processVersion.startsWith("v10")) {
```

Comment:
Wondering if it might be useful to keep the minor version, since AWS seems to reserve the right to update it 🤔 . On one hand, it's more accurate and might reveal why a regression occurred, on the other it doesn't match the aws specified runtime.